stack.yaml

```diff
@@ -2,6 +2,10 @@ resolver: lts-13.29
 
 extra-deps:
   - diff-parse-0.2.1
+  - hlint-2.2.2
+  - ghc-lib-parser-8.8.0.20190723  # for hlint
+  - haskell-src-exts-1.21.0 # for hlint
```

Comment:
Heh, extra dependency for now... As I recall, `hlint` should get rid of this heavy package and we need to remember to remove it from here when we update to a newer `hlint`.